### PR TITLE
fix: upgrade Operator base image to fix vulnerabilities

### PR DESCRIPTION
**Pull request by ivanstanev**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

Upgrade Operator base image to fix vulnerabilities.
Resolves https://github.com/snyk/kubernetes-monitor/issues/574.

### CODE REVIEW

File: snyk-operator/build/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/operator-framework/helm-operator:v0.15.1
+FROM quay.io/operator-framework/helm-operator:v1.1.0
```
Comment (agranado2k) [inline snyk-operator/build/Dockerfile:1]: I'm curious if is it possible it break any integration test or the deployment process to create the Operator? I think it should not... however...

**Review by agranado2k — COMMENTED** [review]
I minor comment

File: snyk-operator/build/Dockerfile (hunk 1 above)
Comment (ivanstanev) [reply snyk-operator/build/Dockerfile:1]: Let's find out 🙂 It's all good to revert until we publish to GitHub so we have plenty of time to detect issues

**Review by agatakrajewska — APPROVED** [review]

**Review by shaimendel — DISMISSED** [review]

**Review by agranado2k — APPROVED** [review]
LGTM! 🎉

**Pull request merged**